src/solve.jl

```diff
@@ -35,11 +35,12 @@ function DiffEqBase.__init(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase.
                            qmax = qmax_default(alg),
                            qsteady_min = qsteady_min_default(alg),
                            qsteady_max = qsteady_max_default(alg),
+                           beta1 = nothing,
+                           beta2 = nothing,
                            qoldinit = isadaptive(alg) ? 1//10^4 : 0,
+                           controller = nothing,
```

Comment:
we'd need to level this up to a common kwarg if it goes here.

Reply:
What exactly do you mean by that?
- It needs to be documented (sure it does)?
- It needs to be implemented for other solvers/packages (delay, stochastic, quadrature, linear, nonlinear, ...)?
- Something else?